docs/content/how-to/creating-catalogs.md

```diff
@@ -175,3 +176,185 @@ Using the table option facilitates the convenient definition of Hive table param
+
+> If you are using Hive3, please disable Hive ACID:
+>
+> ```shell
+> hive.strict.managed.tables=false
+> hive.create.as.insert.only=false
+> metastore.create.as.acid=false
+> ```
+
+### Setting Location in Properties
+
+If you are using an object storage , and you don't want that the location of paimon table/database is accessed by the filesystem of hive,
+which may lead to the error such as "No FileSystem for scheme: s3a".
+You can set location in the properties of table/database by the config of `location-in-properties`. See
+[setting the location of table/database in properties ]({{< ref "maintenance/configurations#HiveCatalogOptions" >}})
+
+### Synchronizing Partitions into Hive Metastore
+
+By default, Paimon does not synchronize newly created partitions into Hive metastore. Users will see an unpartitioned table in Hive. Partition push-down will be carried out by filter push-down instead.
+
+If you want to see a partitioned table in Hive and also synchronize newly created partitions into Hive metastore, please set the table property `metastore.partitioned-table` to true. Also see [CoreOptions]({{< ref "maintenance/configurations#CoreOptions" >}}).
+
+### Adding Parameters to a Hive Table
+
+Using the table option facilitates the convenient definition of Hive table parameters.
+Parameters prefixed with `hive.` will be automatically defined in the `TBLPROPERTIES` of the Hive table.
+For instance, using the option `hive.table.owner=Jon` will automatically add the parameter `table.owner=Jon` to the table properties during the creation process.
+
+
+## Creating a Catalog with JDBC Metastore
+
+By using the Paimon JDBC catalog, changes to the catalog will be directly stored in relational databases such as MySQL, postgres, etc.
+
+{{< tabs "jdbc-metastore-example" >}}
+
+{{< tab "Flink" >}}
+
+Paimon JDBC Catalog in Flink needs to correctly add the corresponding jar package for connecting to the database. You should first download JDBC  connector bundled jar and add it to classpath. such as MySQL, postgres
+
+| database type | Bundle Name          | SQL Client JAR                                                             |
+|:--------------|:---------------------|:---------------------------------------------------------------------------|
+| mysql         | mysql-connector-java | [Download](https://mvnrepository.com/artifact/mysql/mysql-connector-java)  |
+| postgres      | postgresql           | [Download](https://mvnrepository.com/artifact/org.postgresql/postgresql)   |
+
+```sql
+CREATE CATALOG my_jdbc WITH (
+    'type' = 'paimon',
+    'metastore' = 'jdbc',
+    -- 'uri' = 'jdbc:mysql://<host>:<port>/<databaseName>',
+    -- 'jdbc.user' = '...', 
+    -- 'jdbc.password' = '...', 
+    -- 'initialize-catalog-tables'='true'
```

Comment:
Do we need this one? What is the scene?

Reply:
> Do we need this one? What is the scene?

@JingsongLi When creating a JdbcCatalog, it will be determined based on this parameter whether to automatically create a table or not

Reply:
When does it not need to be initialized? You added a public API. Please explain why, how to use.

Reply:
Or you can remove this option.